lib/action_controller/serialization.rb

```diff
@@ -27,6 +27,7 @@ def get_serializer(resource, options = {})
       end
       serializable_resource = ActiveModel::SerializableResource.new(resource, options)
       if serializable_resource.serializer?
+        serializable_resource.url_helper = self
```

Comment:
Thanks for this

:-1: We will not include the controller. That would be a **huge** bug and complexity attractor

I think there's an issue for this, but I think the `:context` option should be renamed `:request_context` and be an object named `RequestContext` that has the methods `original_url`, `query_params`, and `url_helper`, though  them's just my thoughts.. but the main this is that we will *not* pass around a controller instance